src/Tools/ExternalAccess/OmniSharp/Internal/PickMembers/OmniSharpPickMembersService.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.PickMembers;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.PickMembers;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.Internal.PickMembers
+{
+    [Shared]
+    [ExportWorkspaceService(typeof(IPickMembersService))]
```

Comment:
💭 This is a host-layer service right?

Reply:
Yes, the existing impl is `VisualStudioPickMembersService`. Should I be setting an explicit layer here?

Reply:
I would have assumed this needs to be marked as "Host" for the service layer, or otherwise this and the other one are both "Default", and I'm not sure which MEF service is going to win there.

Reply:
So I _should_ do this, or I shouldn't? Happy to mark it as host, just need to know I should.

> I'm not sure which MEF service is going to win there.

I would never expect this service to be loaded in the same process as the VS host version, fwiw.

Reply:
Oh, I see -- this is a service that *has* no default implementation -- the only implementation comes from the host. I was worried this was going to conflict with the default impl, but there is no such thing. So this will work, but that said, if you mark it Host it'll be marked the same way like everything else, which sure won't hurt anything. When in Rome...

Reply:
Done.